src/universal/modules/userDashboard/containers/Me/Me.js

```diff
@@ -1,22 +1,29 @@
 import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {cashay} from 'cashay';
+import {getAuthQueryString, authedOptions} from 'universal/redux/getAuthedUser';
 import Me from 'universal/modules/userDashboard/components/Me/Me';
 import requireAuth from 'universal/decorators/requireAuth/requireAuth';
-import {connect} from 'react-redux';
 
-const mapStateToProps = state => ({
-  authToken: state.authToken
-});
+const mapStateToProps = (state, router) => {
+  return {
+    authToken: state.authToken,
+    location: router.location.pathname,
+    user: cashay.query(getAuthQueryString, authedOptions).data.user
+  };
+};
 
 @connect(mapStateToProps)
 @requireAuth
 // eslint-disable-next-line react/prefer-stateless-function
 export default class MeContainer extends Component {
   static propTypes = {
+    location: PropTypes.string,
     user: PropTypes.object
   };
 
   render() {
-    const {user} = this.props;
-    return <Me user={user}/>;
+    const {location, user, ...props} = this.props;
```

Comment:
more evil stinky bad... passing down `location` is a precursor to an anti pattern.